install.sh

```diff
@@ -18,7 +18,7 @@ print_usage_info()
     echo "yes | ./install.sh"
 }
 
-GCC_VERSION=11
+GCC_VERSION=12
 GASNET_VERSION="2021.9.0"
```

Comment:
How about also updating GASNet to the current 2022.3.0 release?

Reply:
@bonachea done.  In case it's useful, you can make a suggested edit in a comment by clicking the boxed "+/" button and then I can click to accept and apply the change.